velox/functions/CoverageUtil.h

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <string>
+#include <unordered_set>
+#include <vector>
+
+std::vector<std::string> readFunctionNamesFromFile(const std::string& filePath);
+
+std::string toFuncLink(const std::string& name, const std::string& domain = "");
```

Comment:
Is this function used outside of the the CoverageUtil.cpp? If not, let's move it to the .cpp file. Please, go over all functions / classes in this .h file and move the ones that are not used outside of the .cpp file there.

Reply:
Thanks, just fixed it. now the header file looks nicer.